tests/staggered_eigensolve_test.cpp

```diff
@@ -192,24 +192,75 @@ std::vector<double> eigensolve(test_t test_param)
 
   if (!enable_testing || (enable_testing && getVerbosity() >= QUDA_VERBOSE)) display_test_info(eig_param);
 
+  // Gauge Smearing Routines
+  if (gauge_smear) {
+    quda::host_timer_t host_timer;
+    host_timer.start(); // start the timer
+
+    QudaGaugeObservableParam *obs_param = new QudaGaugeObservableParam[gauge_smear_steps / measurement_interval + 1];
```

Comment:
This needs a corresponding `delete[]`

Reply:
Or we could use a `std::vector<QudaGaugeObservableParam>` instead, I believe?

Reply:
196cef438